source/_posts/Rails-Starter-Template.md

```diff
@@ -0,0 +1,209 @@
+---
```

Comment:
Use keywords throughout your article

Reply:
Article should be based on content, not keywords